### PR TITLE
Update view-only-meeting-experience.md

### PR DESCRIPTION
**Pull request by chbalaki**

### CODE REVIEW

File: Teams/view-only-meeting-experience.md [hunk 1]
```diff
@@ -124,6 +124,7 @@ The view-only attendee won't be able to experience the following options in meet
 
 - View-only attendees will only be able to see Live Captions on Desktop and Web. Only English Captions are supported at this time.
 - View-only attendees cannot register for Webinars.
+- View-only attendees will not have Information Barrier support. If you need Information Barrier support in your organization, please disable this feature.
```
Comment (JohanFreelancer9) [inline Teams/view-only-meeting-experience.md:127]:
```suggestion
- View-only attendees will not have Information Barrier support. If you need Information Barrier support in your organization, you should disable this feature. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @chbalaki, kindly commit the suggested change. Many thanks.

**Comment by chbalaki** [comment]
Just did - thanks so much!

**Comment by chbalaki** [comment]
Also I'm unsure how to push the final changes, do I just 'Close with comment' on my UI?

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by JohanFreelancer9** [comment]
> Also I'm unsure how to push the final changes, do I just 'Close with comment' on my UI?

No, please don't close the PR. It will be merged once the doc owner or manager approves it.

**Comment by chbalaki** [comment]
Thanks @JohanFreelancer9 - this has been open for about 4 days. Who is the owner of this document? Can we nudge them?

**Comment by JohanFreelancer9** [comment]
> Thanks @JohanFreelancer9 - this has been open for about 4 days. Who is the owner of this document? Can we nudge them?

@scanum, please assist.

**Comment by chbalaki** [comment]
@scanum, @SerdarSoysal, can we please approve this change? This documentation is causign customer confusion without this update. Thanks!

**Comment by scanum** [comment]
@tiburd Hi Tina! Could you kindly help with this?

**Comment by tiburd** [comment]
@chbalaki 
Hi,

This pull request was opened in the public repo. PMs and content developers should work in the private repo, as described in [the authoring guide](https://review.docs.microsoft.com/en-us/Office-Authoring-Guide/office-docs-authoring-overview?branch=main).

At this time, we're proceeding with this PR, but please open future PRs in the private repo.

Because you are the ms.reviewer for the article, we are merging this PR without additional approval.

Thanks!

cc: @mkbond007 @scanum

**Pull request merged**